ztp/siteconfig-generator/siteConfig/siteConfigBuilder.go

```diff
@@ -193,21 +191,41 @@ func (scbuilder *SiteConfigBuilder) getExtraManifest(dataMap map[string]interfac
 				dataMap[filename] = value
 			}
 		} else {
-			manifestFile, err := scbuilder.fHandler.ReadSourceFile(filePath)
+			manifestFile, err := ReadFile(filePath)
 			if err != nil {
 				return dataMap, err
 			}
 			manifestFileStr := string(manifestFile)
 			dataMap[file.Name()] = manifestFileStr
 		}
 	}
+
+	// Adding End User Extra-manifest
+	files, err = GetFiles(clusterSpec.ExtraManifestPath)
+	if err != nil {
+		return dataMap, err
+	}
+	for _, file := range files {
+		if file.IsDir() {
+			continue
+		}
+
+		filePath := clusterSpec.ExtraManifestPath + "/" + file.Name()
```

Comment:
Because we're adding content provided by the end-user, we should probably log this - Print out at least the filenames that we're adding.